### DISCUSSION TITLE
Curriculum Materials Site

### DISCUSSION THREAD

**Discussion by Jlester92** [TEALSK12/introduction-to-computer-science#376] [General]
The previous teacher to my class sent me a link the the Curriculum Materials Site: https://teals.sharepoint.com/curriculum/Shared%20Documents/Forms/AllItems.aspx

The username and password she gave me does not work anymore.  Is there an updated password or way to login?

**Comment by degriff** [comment]
Did you try the credentials at [https://www.tealsk12.org/curriculum/](https://www.tealsk12.org/curriculum/)?

**Reply by Jlester92** [reply]
I don't  have permissions to that site.

**Comment by aspiece** [comment]
> The previous teacher to my class sent me a link the the Curriculum Materials Site: https://teals.sharepoint.com/curriculum/Shared%20Documents/Forms/AllItems.aspx
> 
> The username and password she gave me does not work anymore. Is there an updated password or way to login?

Hello @Jlester92, you can find the credentials in the TEALS Dashboard or reach out to your local regional manager and they can give you this information. We would prefer not to post the credentials publicly.

@degriff I would recommend using the curriculum materials sharepoint to access answer keys. This is where we have the most up to date answer keys for things. We have moved to keeping everything in one place for all of the TEALS Curriculum.

Andrew

**Reply by degriff** [reply]
Until today, I had not gone to the Sharepoint repository for nearly 2 years and solely relied on this GitHub repository.  The only reason I went to Sharepoint is that my teacher needs to create a written exam and I shared the quizzes so that he would not need to start from scratch.

**Comment by Jlester92** [comment]
@aspiece I don't have alot of information on my dashboard.  I was not given a whole lot of guidance when I took over this call other than here is the website and some materials.  I assume that I have to complete the school application, etc. to get access to these materials?

**Reply by DBashirTEALS** [reply]
Hi @Jlester92 is your school a part of the TALS program?  If so, can I ask what the name of your school is?  I can connect you with your Regional Manager in order to get this straightened out.  

Please let me know.  Thanks!

Dr. Bashir

**Reply by Jlester92** [reply]
As far as I know we are not.  Kennedy High School in Cedar Rapids, IA

**Reply by DBashirTEALS** [reply]
@Jlester92 do you have an email I can reach you at?  I would like to take this conversation offline, so I am able to assist you a bit more.

**Reply by Jlester92** [reply]
jlester@crschools.us
